Vagrantfile

```diff
@@ -43,14 +45,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         echo "Installing required packages"
         set -x
         sudo apt-get update
-        sudo apt-get install --yes git python3-pip python3-venv rabbitmq-server amqp-tools mosquitto mosquitto-clients httpie socat
+		sudo apt-get upgrade
+        sudo apt-get install --yes git python3-pip python3-venv rabbitmq-server amqp-tools mosquitto mosquitto-clients httpie socat mariadb-server asterisk ffmpeg
 
         # Git settings for `root`.
         sudo git config --global pull.ff only
 
         # Git settings for `vagrant`.
         su vagrant -c "git config --global pull.ff only"
     SHELL
+	
+	# Setup MariaDB
+    machine.vm.provision :shell, privileged: true, inline: <<-SHELL
+        debconf-set-selections <<< "mysql-server mysql-server/root_password password secret"
+		debconf-set-selections <<< "mysql-server mysql-server/root_password_again password secret"
+		#apt-get -y install mysql-server
+		mysql -e "CREATE DATABASE secpi;"
+		mysql -e "grant all privileges on secpi.* TO 'secpi'@'localhost' identified by 'secret';"
+		mysql -e "grant all privileges on secpi.* TO 'secpi'@'%' identified by 'secret';"
```

Comment:
```suggestion
		mysql -e "grant all privileges on 'secpi-testdrive.*' TO 'secpi'@'localhost' identified by 'secret';"
		mysql -e "grant all privileges on 'secpi-testdrive.*' TO 'secpi'@'%' identified by 'secret';"
```

Reply:
Sorry for the wrong hint. For properly quoting the database name, backticks should be used. See 5237e3aa.